apps/landing-page/pages/blog/topology-plugin-coming-soon.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Topology plugin coming soon to a Backstage near you!
+date: 2023/03/06
+description: Visualize your k8s workloads in a new and efficient way!
+---
+
+import { Authors } from 'ui/components';
+
+<Authors authors={['_div_gupta']} />
+
+# Topology plugin coming soon to a Backstage near you!
+
+The Janus community is thrilled to share details about our Backstage Topology Plugin. This powerful new tool simplifies the process of visualizing k8s workloads of your Backstage services. With this plugin, developers can get a clear and concise overview of their application's structure and workload status. This eliminates the stress and cognitive overload that often comes with working with Kubernetes.
+
+![Topology plugin coming soon!](/images/blog/topology-plugin-coming-soon/topologyplugin.png)
```

Comment:
```suggestion
![Topology plugin coming soon!](/images/blog/topology-plugin-coming-soon/topology-plugin.png)
```

You will also need to rename the image file. Its generally best practice to name files in file case.